members/views.py

```diff
@@ -80,3 +81,16 @@ def post(self, request):
             return JsonResponse({"message": "EXIST_MEMBER"}, status=400)
 
         return JsonResponse({"message": "SUCCESS"}, status=200)
+
+class MemberView(View):
+    @login_decorator
+    def get(self, request):
+        info = {
```

Comment:
```suggestion
        information = {
```
wecode에서는 단어를 줄여 쓰는 것을 지양하고 있습니다.

Reply:
해당 사항 반영했습니다!